test/adapters/http_server.spec.js

```diff
@@ -81,6 +81,41 @@ describe('read http_server', function() {
         });
     });
 
+    it('handles delayed point arrival', function() {
+        var programFinish, program;
+        var body = { 'name': 'ted', 'age': 53, 'weight': 100};
+
+        return compile_juttle({
+            program: 'read http_server -every :100ms: -port ' + port
+        })
+        .then(function(prog) {
+            program = prog;
+            programFinish = run_juttle(program);
+        })
+        // Wait a while before posting the data
+        .delay(500)
```

Comment:
How? The idea here is to add some delay during which the adapter is idle, then send the post, then give the events some time to propagate, then get the response. I don't see a cleaner solution using retry, but if you have something in mind then I'd be open to it.